docs/cloud/rahti2/images/Using_Rahti_2_integrated_registry.md

```diff
@@ -39,3 +41,41 @@ Go to the Images section, make sure the option "Deploy images from an image stre
 Finally select the new image.
 
 ![Use cached image](../../img/use_cached_image.png)
+
+## Access Control for the Rahti 2 Integrated Registry
+
+Rahti 2 allows fine-grained control over access to the integrated image registry, enabling management of access based on [user authentication](https://docs.openshift.com/container-platform/4.15/authentication/understanding-authentication.html#rbac-groups_understanding-authentication).
+
+### 1. **Anonymous Access** (`system:anonymous`)
```

Comment:
I'm a bit confused between case 1 and 2 (since I know nothing of this - as many our customers I guess): Do I need to do both of them? I mean add both anonymous and unauthorized to registry-viewer? Or only one of them? If only one of them, what would be the difference? I mean how anonymous would authenticate? Is there some project related password or token to be used?

I'm sure the technical instructions are right here, but the context could be opened up a bit. 

Case three is quite clear I think.

Reply:
They are the same, anonymous is for users and unauthenticated is super set that includes anonymous + other stuffs. unauthenticated can be used for scripts or automation tools. I have stated the use cases when and what to use, other wise either one would be fine for anonymousity.